src/main/java/com/notably/model/viewstate/ViewStateModelImpl.java

```diff
@@ -12,10 +17,13 @@
 public class ViewStateModelImpl implements ViewStateModel {
 
     private StringProperty input;
+    private Property<Optional<String>> responseTextProperty;
+
     private final BooleanProperty helpOpen;
 
     public ViewStateModelImpl() {
         this("", false);
+        responseTextProperty = new SimpleObjectProperty<>(Optional.empty());
     }
 
     private ViewStateModelImpl(String initialInput, boolean initialHelpBool) {
```

Comment:
Better yet, you create your constructors as such:

```java
public ViewStateModelImpl() {
  this("", Optional.empty(), false);
}

private ViewStateModelImpl(String initialInput, Optional<String> initialResponseText, boolean initialHelpOpen) {
  this.input = new SimpleStringProperty(initialInput);
  this.responseTextProperty = new SimpleObjectProperty<>(initialResponseText);
  this.helpOpen = new SimpleBooleanProperty(initialHelpOpen);
}
```

This is to make it such that your constructors are consistent

Reply:
coolios sickerino thanks!